### PR TITLE
Make explorer work with localstack

### PR DESCRIPTION
**Pull request by andres-lowrie**
You'll need to rebuild the bootstrap and s3-explorer services to get these changes going in your local environment

> If you're using the `rake` provided tmux session then exit out of the session (by closing each window/pane) and then do `rake clean` to remove all the temporary files and what not

To rebuild the images you should be able to run the following commands from the root level dir of the back-end project

```shell
docker-compose build s3explorer && docker-compose build bootstrap
```

then you can do `rake dev` to start up all the session again

### CODE REVIEW

**Comment by andres-lowrie** [comment]
NOTE: Once we merge this in, we need to update the submods on the main repo... I'll take care of that once this gets merged in

**Review by runrog — APPROVED** [review]
😭 +1

File: index.html [hunk 1]
```diff
@@ -25,13 +25,15 @@
...
         <link rel="stylesheet" href="//cdn.datatables.net/plug-ins/f2c75b7247b/integration/bootstrap/3/dataTables.bootstrap.css">
+        <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/default.min.css">
         <style type="text/css">
             #wrapper { padding-left: 0; }
             #page-wrapper { width: 100%; padding: 5px 15px; }
             #tb-s3objects { width: 100% !Important; }
             a { color: #00B7FF; }
             body { font: 14px "Lucida Grande", Helvetica, Arial, sans-serif; }
             td { font: 12px "Lucida Grande", Helvetica, Arial, sans-serif; }
+            code { display: block; white-space: pre-wrap; }
```
Comment (runrog) [inline index.html:36]: 🤦‍♂️ "css"

Comment (andres-lowrie) [reply index.html:36]: You like them skills boy!!!!

**Pull request merged**